README.md

```diff
@@ -119,6 +126,7 @@ Resources
 ---------
 
 - [ARD (Apple Remote Desktop)](./documentation/resource_ard.md)
```

Comment:
These links should be updated with the full address since they are posted on the Chef Supermarket as well.